dockerfiles/databricks-sql-cli.Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.11-slim
```

Comment:
don't know if we still want to produces this kind of image, cf https://kestra-io.slack.com/archives/C04HTFM3VL6/p1747239026022299